thefuck/main.py

```diff
@@ -1,8 +1,6 @@
 # Initialize output before importing any module, that can use colorama.
 from .system import init_output
 
-init_output()
```

Comment:
This one is necessary. Looks like we'll need `# noqa: E402` in every of the following import lines.